test/grabURITest.coffee

```diff
@@ -0,0 +1,222 @@
+# Copyright (C) 2016 The Firehose Project
+#
+# This software may be modified and distributed under the terms
+# of the MIT license.  See the LICENSE file for details.
```

Comment:
Better to point at a URI instead of a license file. The header is assuming that the file has been broken out of context, so the license file is likely not there. Below is my std header for MIT:
```
# © YEAR LasLabs Inc.
# License MIT (https://opensource.org/licenses/MIT).
```